operate/quickstart/choose_staking.py

```diff
@@ -131,6 +131,30 @@ def get_staking_contract_metadata(self: "StakingHandler", program_id: str) -> Di
                 "description": program_id,
             }
 
+    def get_staking_slots_count(self: "StakingHandler", program_id: str) -> int:
```

Comment:
can we add this algorithm in `get_staking_contract_metadata` and just return the available staking slots along with `name` and `description` there? Because right now, we're calling `_get_staking_token_contract` twice (which internally makes many RPC calls), and that's making it very slow.